src/translations/en.json

```diff
@@ -5571,10 +5571,16 @@
         },
         "core-config": {
           "intro": "Hello {name}, welcome to Home Assistant. How would you like to name your home?",
+          "intro_core": "We will be setting up some core configuration for your Home Assistant instance now, you can always change this later in the configuration panel.",
```

Comment:
```suggestion
          "intro_core": "We will be setting up the basics together. You can always change this later.",
```

Reply:
Not sure if this rephrase is a bit too radical. My aim is to reduce the word count and remove technical wording like core configuration and configuration panel. While this info is correct, I'm not sure the user needs to know this at this point.

Reply:
May be "You can always change this later in the settings" ?

Reply:
yes, I like that.